src/QuizController.js

```diff
@@ -23,24 +24,43 @@ function getRandomInt(max) {
     return Math.floor(Math.random() * Math.floor(max));
 }
 
+const NUM_EXERCISES = 10;
+
 class QuizController extends Component {
     constructor() {
         super()
 
         this.state = {
-            numExercises: 10,
+            numExercises: NUM_EXERCISES,
```

Comment:
It might make sense to consider whether this is state or a prop or none of the above.

Reply:
Mmhmmm.  I'm guessing some of my other refactors will force me to choose a good answer here so I'll wait until I get there.